### PR TITLE
TAO-455 : use CDATA instead of weird  escaping

### PR DESCRIPTION
**Pull request by clawfire**
I test few interaction to see if it's cause any trouble when we enter a response. all looks good. Need more intense testing from our tester.

### CODE REVIEW

**Comment by krampstudio** [comment]
To be sure it has no impact on the serialization, you should add a test case in `test/QtiParsingTest.php` with a sample that has `CDATA` in the `correctResponse` tag and see if the `responseDeclaration` gives you the correct value.
@ssipasseuth can lead you on adding a test case

**Comment by clawfire** [comment]
@krampstudio done

File: test/QtiParsingTest.php [hunk 1]
```diff
@@ -31,64 +31,93 @@
...
+     * @author Thibault Milan <thibault.milan@vesperiagroup.com>
+     */
+    public function testFileParsingCDATA(){
+        $file = dirname(__FILE__).'/samples/xml/cdata/item.xml';
+
+        $qtiParser = new Parser($file);
+        $qtiParser->validate();
+
+        if(!$qtiParser->isValid()){
+            echo $qtiParser->displayErrors();
```
Comment (krampstudio) [inline test/QtiParsingTest.php]: I would rather use
```php
$this->fail($qtiParser->displayErrors());
```

Comment (clawfire) [inline test/QtiParsingTest.php]: I was just reusing the code just before my test :D But thanks, I'll change that just in a sec

**Comment by clawfire** [comment]
Can't you test if when you save an item, the correct XML is generated **AND** when you leave and come back in authoring, the response is loading in a proper way please ?

**Comment by clawfire** [comment]
Ok I'll adress the ckeditor bug in a separate PR, if this one is ok for you can we merge it ?

**Pull request merged**